src/BusinessService.ts

```diff
@@ -65,6 +65,14 @@ export type Business = {
    * External is true when a business is created by external provider.
    */
   external: boolean;
+  /**
+   * The website URL of the business
+   */
+  websiteUrl: string;
+  /**
+   * The department name of the business
+   */
+  department: string;
```

Comment:
Are these fields really required? Or should they be optional / nullable?

Reply:
They should be required, at least by the ticket description. @nevalla ?